src/js/utils/ValidatorUtil.js

```diff
@@ -1,11 +1,116 @@
 var ValidatorUtil = {
+
+  /**
+   * Check if the given string is an e-mail
+   *
+   * @param {string} email - The e-mail string to test
+   * @returns {boolean} - Returns true if the string is an e-mail
+   */
   isEmail: function (email) {
     return email != null &&
       email.length > 0 &&
       !/\s/.test(email) &&
       /.+@.+\..+/
       .test(email);
+  },
+
+  /**
+   * Check if the given string is a valid cron component
+   *
+   * @param {string} component - The cron component to test
+   * @param {number} numMin - The minimum value accepted in the numerical components
+   * @param {number} numMax - The maximum value accepted in the numerical components
+   * @param {array} discreteValues - Other valid discreet values
+   * @returns {string} - Returns null if test passes otherwise return the error string
+   */
+  testCronComponent: function (component, numMin, numMax, discreteValues = []) {
```

Comment:
I chose `testCRONComponent` instead of `isCRONComponent` because instead of returning a boolean value it provides the error description.

Naming aside, I separated the functions to a new util file.